extensions/laravel-nova/src/nova.tsx

```diff
@@ -0,0 +1,197 @@
+import {
+  getPreferenceValues,
+  ActionPanel,
+  List,
+  CopyToClipboardAction,
+  OpenInBrowserAction,
+  showToast,
+  Toast,
+} from "@raycast/api";
+import { useEffect, useMemo, useState } from "react";
+import algoliasearch from "algoliasearch";
+import striptags from "striptags";
+
+const APPID = "BH4D9OD16A";
+const APIKEY = "5aa44fede3f10262000a8c4f046033d5";
```

Comment:
is it alright to hardcode the APIKEY here? Won't there be any limitation for usage?

Reply:
Not sure at all if I'm honest. I'm following the same pattern seen in other algolia based extensions / even in an Alfred workflow that I was using prior to Raycast.

https://github.com/raycast/extensions/blob/fd5ead772a1341cd2f2f6fcfa4e51eda7abcd9ab/extensions/laracasts/src/laracasts.tsx#L6-L8

https://github.com/raycast/extensions/blob/fd5ead772a1341cd2f2f6fcfa4e51eda7abcd9ab/extensions/laravel-docs/src/index.tsx#L39-L41

https://github.com/techouse/alfred-nova-docs/blob/master/bin/src/constants/config.dart#L7-L10